google/cloud/security/scanner/audit/base_rules_engine.py

```diff
@@ -50,7 +50,7 @@ def __init__(self,
         self.full_rules_path = rules_file_path.strip()
         self.snapshot_timestamp = snapshot_timestamp
 
-    def build_rule_book(self):
+    def build_rule_book(self, global_configs):
```

Comment:
Right, good question.  When I was at this, my thought is that the ```configs``` variable is important enough to be explicit, rather than abstracting it as a kwarg.